src/index.ts

```diff
@@ -77,6 +77,12 @@ export * from './components/tooltip';
 export * from './components/user_account_menu';
 export * from './components/video_player';
 
+export * from './components/input';
+export { InputText } from './components/input/input_text';
+export { SearchBar } from './components/search_bar';
+// once icons are in their own package this will be removed
+export * from './components/icons';
```

Comment:
Icons are components too; I'd prefer to avoid ninety billion PRs and release and npm installs required when we make changes.

Reply:
Good, should anything change, then there'll only be a few so we can deal with that.

Reply:
My point being that I disagree that icons should be a different package.

Reply:
Fine, we'll discuss that when we get there :) This PR still contains the icons in case you had missed that.